### ISSUE TITLE
[+orion] Bug 'do you like yourself'

### ISSUE THREAD

**Issue by orion-botx** [sugaroidbot/sugaroid#63]
The following answer was a given by sugaroid, and is considered as a bug. Please 
	manually review this issue, and close it if it looks unnecessary.

	user: do you like yourself 
    sugaroid: Yes. I guess I like to like

**Comment by sreyasaju** [comment]
@srevinsaju @TheDarkDrake,  do you have any suggestions on the reply sugaroid should give on `"do you like yourself"`

**Comment by srevinsaju** [comment]
hmm

**Comment by TheDarkDrake** [comment]
Hmmm indeed

**Comment by srevinsaju** [comment]
yes,, indeed

**Comment by srevinsaju** [comment]
@sreyasaju for insight, read this: https://www.quora.com/Do-you-like-yourself-1

**Comment by TheDarkDrake** [comment]
Wait, wasn't the reply to "do you like yourself" different

**Comment by srevinsaju** [comment]
> Wait, wasn't the reply to "do you like yourself" different

different?

**Comment by srevinsaju** [comment]
this was in the telegram group tho

**Comment by sreyasaju** [comment]
like to like?

**Comment by TheDarkDrake** [comment]
Wait, let me check

**Comment by srevinsaju** [comment]
oh yes

**Comment by srevinsaju** [comment]
Thats right, I am not sure how it came

**Comment by TheDarkDrake** [comment]
At different times, sugaroid gives different answers. Sometimes it understands, sometimes it doesn't. Sometimes it gives one answer, sometimes different

**Comment by srevinsaju** [comment]
For 
* Do you like to like: Give a list of answers and choose a random one
* Do you like yourself: Read that previous Quora answer

**Comment by srevinsaju** [comment]
Does it sound right?

**Comment by sreyasaju** [comment]
wait, should sugaroid  `like the user more than itself` or
 should it `like itself more than the user`

**Comment by srevinsaju** [comment]
> wait, should sugaroid `like the user more than itself` or
> should it `like itself more than the user`

good question

**Comment by srevinsaju** [comment]
![image](https://user-images.githubusercontent.com/48695438/107149201-5a740400-6968-11eb-913c-b8f0a649f62b.png)

A group of people have voted for "Sugaroid Itself", as sugaroid is not a bot, but a bot which is trying to act like real people. As it needs Human Traits, its important to like oneself, and we are not comparing it with other people. For the question, "Do you like yourself", the answer should be yes. You should not say, just yes. But a bunch of philosophical statements which seem to agree with the accepted idea.

**Comment by sreyasaju** [comment]
hmm.. makes sense :)

**Comment by TheDarkDrake** [comment]
Yep, perfect. Now, the response as a whole. A response should be made having those philosophical statements
